src/helpers.c

```diff
@@ -324,3 +324,13 @@ bool file_is_debuginfo(const char *path)
 	free(src);
 	return ret;
 }
+
+bool comp_ghosted(struct file *file1, struct file *file2)
+{
+	return (file1->is_ghosted && file2->is_ghosted);
```

Comment:
Really should check if `file1` or `file2` is null.

Reply:
Since this is only used in create_update.c it should go in there and not be declared external.

I don't think it should check for file1 and file2 being not null, it is not clear what the return value should be in that case.

Same comment on the name as for comp_deleted

Reply:
+1 on moving (and removing declarations from `swupd.h`) and not checking for `NULL` (the caller must ensure that).